src/lib/constants/defaultModelObjects.ts

```diff
@@ -10,3 +10,30 @@ export const DEFAULT_CATEGORY_COMBO = {
     isDefault: true,
     name: 'default',
 }
+
+export const DEFAULT_CATEGORY = {
+    id: 'GLevLNI9wkl',
+    isDefault: true,
+    name: 'default',
+}
+
+export const DEFAULT_CATEGORYCOMBO_SELECT_OPTION = {
+    categories: [
+        {
+            name: 'default',
```

Comment:
Would it make sense to do:
```suggestion
            ...DEFAULT_CATEGORY,
```
here?